src/fsharp/SyntaxTree.fs

```diff
@@ -180,7 +180,7 @@ type DebugPointAtSwitch =
 
 [<RequireQualifiedAccess>]
 type DebugPointAtSequential =
-    | AlwaysBoth
+    | SuppressNeither
```

Comment:
Oh, I like that change, makes it easier to read!

Reply:
Yes I'm minimizing the diff a bit.  There still a lot of changes to the baselines we need to check carefully, and pin more things down in code gen tests